docs/howto.md

```diff
@@ -39,17 +39,23 @@ int main() {
     ozo::rows_of<std::int64_t, std::optional<std::string>> rows;
 
     // Connection info with host and port to connect to
-    auto conn_info = ozo::make_connection_info("host=... port=...");
+    // This also supports a PostgreSQL connection URI string
+    ozo::connection_info<> conn_info = ozo::make_connection_info("host=... port=...");
```

Comment:
I'm not quite sure about this change.
Could you explain the reason why did you change the `auto` keyword to the specific type?